### PR TITLE
Add All Display Events

### PR DESCRIPTION
**Pull request by hyjeko**
- add heartbeat log
- add all display event stubs

### CODE REVIEW

**Comment by hyjeko** [comment]
@jbeaumont1010 thx for pair and review <3

File: StardewPriceTip/ModEntry.cs [hunk 1]
```diff
@@ -47,10 +62,68 @@ public override void Entry(IModHelper helper)
...
+            // LogHeartBeat("onWindowResized()");
+        }
+
         /*********
         ** Game Loop
         *********/
 
+        private void onUpdateTicked(object sender, UpdateTickedEventArgs e)
+        {
+            // LogHeartBeat("onUpdateTicked()");
```
Comment (hyjeko) [inline StardewPriceTip/ModEntry.cs:124]: This is the method we will probably end up using to see the inventory changes and keep track of what the user has sold.

File: StardewPriceTip/ModEntry.cs [hunk 2]
```diff
@@ -47,10 +62,68 @@ public override void Entry(IModHelper helper)
             helper.Events.Input.ButtonPressed += this.onButtonPressed;
         }
 
+        /*********
+        ** Display
```
Comment (hyjeko) [inline StardewPriceTip/ModEntry.cs:66]: This is the section that holds all the display events

File: StardewPriceTip/ModEntry.cs [hunk 3]
```diff
@@ -177,5 +250,10 @@ private void LogLedger(string message = "")
             this.Monitor.Log($"{message} _ledger: {_ledger.ToString()}", LogLevel.Debug);
             this.Monitor.Log($"{message} Game1: {Game1.player.Money.ToString()}", LogLevel.Debug);
         }
+
+        private void LogHeartBeat(string message = "")
+        {
+            this.Monitor.Log($"LogHeartBeat() {message}", LogLevel.Debug);
```
Comment (hyjeko) [inline StardewPriceTip/ModEntry.cs:256]: This is so I don't have to copy past `this.Monitor` everywhere

File: StardewPriceTip/ModEntry.cs [hunk 4]
```diff
@@ -47,10 +62,68 @@ public override void Entry(IModHelper helper)
             helper.Events.Input.ButtonPressed += this.onButtonPressed;
         }
 
+        /*********
```
Comment (jbeaumont1010) [inline StardewPriceTip/ModEntry.cs:65]: Nice work, Intern. :)

**Review by hyjeko — COMMENTED** [review]
🗣️

**Review by jbeaumont1010 — APPROVED** [review]
Reviewed files. Looks great.  Well done, all. :D

**Pull request merged**